client/src/app/core/repositories/base-repository.ts

```diff
@@ -253,21 +253,28 @@ export abstract class BaseRepository<V extends BaseViewModel, M extends BaseMode
      */
     public async update(update: Partial<M>, viewModel: V): Promise<void> {
         const data = viewModel.getUpdatedModelData(update);
-        const targetClass = this.collectionMapperService.getModelConstructor(viewModel.collection);
-        return await this.dataSend.updateModel(new targetClass(data));
+        this.sendUpdate(data);
     }
 
     /**
      * patches an existing model with new data,
      * rather than sending a full update
      *
      * @param update the update to send
-     * @param viewModel the motion to update
+     * @param viewModel the view model to update
      */
     public async patch(update: Partial<M>, viewModel: V): Promise<void> {
```

Comment:
Can we eliminate `patch` here, or is it too much work?

If so, please add a `@depricated` and just call `update` internally. Note that the id-injection has to be done in the update.

Reply:
Already done ;)